### ISSUE TITLE
Upgrade guidance from 1.0.1 to 1.1

### ISSUE THREAD

**Issue by ragesoss** [understrap/understrap-child#341]
My organization just [switched to a new theme](https://github.com/WikiEducationFoundation/wikiedu.org-theme-2022) based on `understrap-child`, but the project was started from 1.0.1. I'd like to be able to update it to take advantage of some of the improvements in 1.1, but I'm not sure how to go about it.

An upgrade guide, or any sort of guidance, would be really appreciated!

**Comment by bacoords** [comment]
Agreed - more migration guides in the docs would be useful. This is a bigger one because you're essentially going from BS4 to BS5 if you go to 1.1, so any custom HTML will need to be modified based on the [migration guide](https://getbootstrap.com/docs/5.1/migration/). 

Here's the basics off the top of my head, but I'll make a note to dig in deeper:
- Review functions.php, there's a few new functions in there.
- Remove package.json and package-lock.json
- Remove the `node_modules` folder
- Copy package.json over from the 1.1+ release
- Make sure your on node v16 and run `npm install`
- Check that you have the BS5 sass files in `/src/sass/assets/`, see if you need to update any variables or custom SCSS and attempt to compile CSS
- Make sure you have the correct BS5 JS files in `/src/js/` and try to compile JS
